host-bin/mount-chroot

```diff
@@ -247,6 +247,8 @@ $passphrase" | ecryptfs-rewrap-passphrase "$wrappedfnek" -
             error 1 "Failed to decrypt $NAME."
         fi
 
+        keyctl link @u @s
```

Comment:
We should add `keyctl new_session >/dev/null` before this line, and I think that'll solve the concern. No need to link the current keyring here (we don't need it in this process).